### PR TITLE
Upgrade some vulnerable dependencies

### PR DESCRIPTION
**Pull request by kadary**
Some direct or transitive deps of goham are vulnerable to multiple attacks:

- https://app.snyk.io/vuln/SNYK-JAVA-ORGAPACHETIKA-72870 (#56) 
- https://app.snyk.io/vuln/SNYK-JAVA-OGNL-30474 (#57)
- https://app.snyk.io/vuln/SNYK-JAVA-COMGOOGLEGUAVA-32236 (#58)
- https://app.snyk.io/vuln/SNYK-JAVA-ORGAPACHETIKA-72437 (#56)

This PR fix those issues by simply upgrading their versions to a more secure one.

### CODE REVIEW

**Comment by aurelien-baudet** [comment]
Thank you for you PR. I will update minimum Java version to Java 8.
In the same time, I will also update all dependencies.

**Comment by kadary** [comment]
Hello,
This will be cool.
And recall that I can do it if you want.

**Comment by aurelien-baudet** [comment]
OK for the help. Here is what I plan to do :
- upgrade minimum version to java 8 (#60) 
- update all dependencies to latest version (#56 #60) 
- add tests to ensure that ogham works with Java 9 10 11 and 12
- update support for spring boot 2
 - mainly add tests to ensure that ogham is compatible with every spring boot version
 - improve integration of freemarker and thymeleaf provided by spring (#38) 
 - add integration of mustache and groovy templates
- release v3.3.0

I can create several distinct issues to track this work and we can work together on it by taking issues. I will also create a dedicated branch for that.
Are you OK ?

**Comment by kadary** [comment]
Hello,

Yes of course. Just create the issues and you can directly assign some to me.

**Comment by aurelien-baudet** [comment]
Issues are created and assigned.
The branch is refactoring/upgrade/java8-and-deps

**Comment by kadary** [comment]
A new PR has been created on the specified branch: https://github.com/groupe-sii/ogham/pull/73

File: ogham-template-thymeleaf/pom.xml [hunk 1]
```diff
@@ -24,6 +24,18 @@
...
+				<exclusion>
+					<artifactId>ognl</artifactId>
+					<groupId>ognl</groupId>
+				</exclusion>
+			</exclusions>
+		</dependency>
+		<dependency>
+			<artifactId>ognl</artifactId>
+			<groupId>ognl</groupId>
+			<version>3.0.21</version>
```
Comment (aurelien-baudet) [inline ogham-template-thymeleaf/pom.xml]: Please use a property for the version number like for all dependencies

**Pull request closed without merging**